### PR TITLE
#761 contact administrator via'zvorotniy zvyazok'page1

### PR DESCRIPTION
**Pull request by OlesiaBarannik**

### CODE REVIEW

File: BackEnd/utils/administration/send_email_feedback.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+EMAIL_CONTENT_SUBTYPE = "html"
+
+
+def send_email_feedback(user_email, message, category):
+    """Function for sending feedback letters to the administrator and the user."""
+    context = {
+        "category": category,
+        "message": message,
+        "user_email": user_email,
+    }
```
Comment (YanZhylavy) [inline BackEnd/utils/administration/send_email_feedback.py:16]: The variable 'protocol', which is used in constructing the URL for the img tag with the CraftMerge logo in the email, is missing, which might cause it not to be displayed."

Comment (OlesiaBarannik) [reply BackEnd/utils/administration/send_email_feedback.py:17]: Done! Thank you

**Review by Lvyshnevska — APPROVED** [review]

**Review by KuzykY — APPROVED** [review]

File: FrontEnd/src/components/Contact/Contact.jsx [hunk 2]
```diff
@@ -1,28 +1,201 @@
...
-      </div>
-      <div className={styles['contact__text_container']}>
-        <h2 className={styles['contact__title']}>{contactText.title}</h2>
-        {renderContent(TEXT_CONTENT)}
-      </div>
-    </div>
-  );
+    const [loading, setLoading] = useState(false);
+    const [showModal, setShowModal] = useState(false);
+    const [modalMessage, setModalMessage] = useState('');
```
Comment (popovycholeg) [inline FrontEnd/src/components/Contact/Contact.jsx:26]: nit: Do we need these two states? We can show the modal if `modalMessage` contains text and clean on close.

File: FrontEnd/src/components/Contact/Contact.jsx [hunk 3]
```diff
@@ -1,28 +1,201 @@
...
+
+
+    return (
+        <div className={styles['contact_container']}>
+            <div className={styles['contact__link_container']}>
+                <LinkContainer />
+                <img
+                    className={styles['contact__img1']}
+                    src={`${process.env.REACT_APP_PUBLIC_URL}/img/dots_12x10.png`}
+                    alt="dots_12x10.png"
```
Comment (popovycholeg) [inline FrontEnd/src/components/Contact/Contact.jsx:91]: This alt is not descriptive. Check rules to write alt for images https://abilitynet.org.uk/news-blogs/five-golden-rules-compliant-alt-text

File: FrontEnd/src/components/Contact/Contact.jsx [hunk 4]
```diff
@@ -1,28 +1,201 @@
...
+                            }}
+                            render={({ field, fieldState: { error } }) =>
+                                <Space direction="vertical" style={{ width: '100%', gap: '0px', }}>
+                                    <Select
+                                        placeholder="Оберіть категорію"
+                                        {...field}
+                                        style={{
+                                        width: 400,
+                                        padding: '0px',
+                                        }}
```
Comment (popovycholeg) [inline FrontEnd/src/components/Contact/Contact.jsx:129]: nit: It doesn't seem to be properly formatted

**Review by popovycholeg — APPROVED** [review]

File: FrontEnd/src/components/Contact/Contact.jsx (hunk 2 above)
Comment (OlesiaBarannik) [reply FrontEnd/src/components/Contact/Contact.jsx]: fixed it, thank you

File: FrontEnd/src/components/Contact/Contact.jsx (hunk 3 above)
Comment (OlesiaBarannik) [reply FrontEnd/src/components/Contact/Contact.jsx]: fixed it

File: FrontEnd/src/components/Contact/Contact.jsx (hunk 4 above)
Comment (OlesiaBarannik) [reply FrontEnd/src/components/Contact/Contact.jsx]: fixed, thank you

File: BackEnd/administration/views.py [hunk 5]
```diff
@@ -28,6 +28,11 @@
 from authentication.models import CustomUser
 from profiles.models import Profile
 from .permissions import IsStaffUser, IsStaffUserOrReadOnly, IsSuperUser
+from .serializers import FeedbackSerializer
+from rest_framework.response import Response
+from rest_framework import status
+from rest_framework.views import APIView
```
Comment (YanZhylavy) [inline BackEnd/administration/views.py:34]: These three imports are no longer used in your views, please remove them.

**Review by YanZhylavy — APPROVED** [review]

**Pull request merged**